src/test/java/by/mariayuran/springboot/service/TicketServiceTest.java

```diff
@@ -0,0 +1,116 @@
+package by.mariayuran.springboot.service;
+
+import by.mariayuran.homework.TicketType;
+import by.mariayuran.springboot.entity.Ticket;
+import by.mariayuran.springboot.entity.User;
+import by.mariayuran.springboot.repositories.TicketRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@SpringBootTest
+public class TicketServiceTest {
+
+    @Mock
+    private TicketRepository ticketRepository;
+
+    @InjectMocks
+    @Autowired
+    private TicketService ticketService;
+
+    private Ticket ticket;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+        ticket = new Ticket();
+        ticket.setUser(new User());
+        ticket.setTicketType(TicketType.DAY);
+        ticket.setCreationDate(LocalDate.now());
+    }
+
+    @Test
+    void testCreateTicket_Positive() {
+        when(ticketRepository.save(any(Ticket.class))).thenReturn(ticket);
+        Ticket createdTicket = ticketService.createTicket(ticket);
+        assertNotNull(createdTicket);
+        assertEquals(ticket, createdTicket);
```

Comment:
if you use assertEquls, its not required to use assertNotNull, because you also check it inside  assertNotNull